profile/README.md

```diff
@@ -1,42 +1,12 @@
-- [autowarefoundation/autoware.universe](https://github.com/autowarefoundation/autoware.universe)
-  - Repository for experimental, cutting-edge ROS packages for Autonomous Driving.
-  - Autoware Universe was created to make it easier for researchers and developers to extend the functionality of Autoware Core
-- [autowarefoundation/autoware_launch](https://github.com/autowarefoundation/autoware_launch)
-  - Launch configuration repository containing node configurations and their parameters.
-- [autowarefoundation/autoware-github-actions](https://github.com/autowarefoundation/autoware-github-actions)
-  - Contains [reusable GitHub Actions workflows](https://docs.github.com/ja/actions/learn-github-actions/reusing-workflows) used by multiple repositories for CI.
-  - Utilizes the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) concept.
-- [autowarefoundation/autoware-documentation](https://github.com/autowarefoundation/autoware-documentation)
-  - Documentation repository for Autoware users and developers.
-  - Since Autoware Core/Universe has multiple repositories, a central documentation repository is important to make information accessible from a single place.
-
-### Contribution guidelines
-
-- [There is no formal process to become a contributor](https://github.com/autowarefoundation/autoware-projects/wiki#contributors) - you can comment on any [existing issues](https://github.com/autowarefoundation/autoware.universe/issues) or make a pull request on any Autoware repository!
-  - Make sure to follow the [Contribution Guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
-  - Take a look at Autoware's [various working groups](https://github.com/autowarefoundation/autoware-projects/wiki#working-group-list) to gain an understanding of any work in progress and to see how projects are managed.
-- If you have any questions or ideas, feel free to start a discussion on [GitHub Discussions in autowarefoundation/autoware](https://github.com/autowarefoundation/autoware/discussions).
+The [Autoware Foundation](https://www.autoware.org/) manages the development of Autoware, the world's leading open-source software project for autonomous driving, and sponsors Autoware-based projects all over the world. Autoware is currently being used by over 100 companies on more than 30 vehicles in over 20 different countries worldwide.
 
 ### Useful resources
 
-- [Autoware documentation](https://autowarefoundation.github.io/autoware-documentation/main/) (also [open source](https://github.com/autowarefoundation/autoware-documentation)!)
 - [Autoware Foundation homepage](https://www.autoware.org/)
+- [Autoware Foundation open source projects](https://github.com/autowarefoundation/autoware-projects/wiki#autoware-foundation-open-source-projects)
+- [Overview of Autoware repositories on GitHub](https://github.com/autowarefoundation/autoware#repository-overview)
```

Comment:
[IMO] Just referencing the repository would be better, considering the maintenance.

```suggestion
- [Autoware main repository](https://github.com/autowarefoundation/autoware)
```

Reply:
That is a fair point.